pkg/compactor/syncer_metrics.go

```diff
@@ -83,6 +89,15 @@ func (m *aggregatedSyncerMetrics) gatherThanosSyncerMetrics(reg *prometheus.Regi
 	m.metaSyncFailures.Add(mfm.SumCounters("blocks_meta_sync_failures_total"))
 	m.metaSyncDuration.Add(mfm.SumHistograms("blocks_meta_sync_duration_seconds"))
 
+	for _, met := range mfm["blocks_meta_synced"].GetMetric() {
```

Comment:
Is the purpose of this passage to increment the gauge? More evidence that this should be a counter. :)